tests/usage.test.ts

```diff
@@ -50,59 +50,57 @@ const schemas = [
     schema: preApplicationSchema,
     examples: getJSONExamples<PreApplication>('preApplication'),
   },
-  // {
-  //   name: 'PrototypeApplication',
-  //   schema: prototypeApplicationSchema,
-  //   examples: getJSONExamples<PrototypeApplication>('prototypeApplication'),
-  // },
+  {
+    name: 'PrototypeApplication',
+    schema: prototypeApplicationSchema,
+    examples: getJSONExamples<PrototypeApplication>('prototypeApplication'),
+  },
 ];
 
 describe.each(schemas)('$name', ({schema, examples}) => {
   const validator = new Validator();
 
   describe("parsing using the 'jsonschema' library", () => {
-    describe.each<Application | PreApplication>(examples)(
-      '$data.application.type.description',
-      example => {
-        test('accepts a valid example', async () => {
-          const result = validator.validate(example, schema as Schema);
-
-          expect(result.errors).toHaveLength(0);
-        });
-
-        test('rejects an invalid example', () => {
-          const invalidExample = {foo: 'bar'};
-          const result = validator.validate(invalidExample, schema as Schema);
-
-          expect(result.errors).not.toHaveLength(0);
-        });
-      },
-    );
+    describe.each<Application | PreApplication | PrototypeApplication>(
+      examples,
+    )('$data.application.type.description || $applicationType', example => {
```

Comment:
This `||` isn't perfect, but it's easy enough to read in the test logs for now? 

Was fumbling with this syntax and how to more elegantly coalesce values - feedback welcome!